docker/importer/volume/fhirConfig.js

```diff
@@ -0,0 +1,101 @@
+'use strict'
+
+const fs = require('fs')
+const https = require('https')
+const http = require('http')
+const path = require('path')
+const { spawn } = require('child_process')
+
+const resourcesPath = path.resolve(__dirname, 'fhirResources')
+
+const downloadResources = (cbk) => {
```

Comment:
@de-laz What is `cbk`?

Reply:
@MattyJ007 The correct standard name used is usually `cb` for a `callback` function. I could revert this to just `callback` for clarity.

Reply:
@MattyJ007  Alternatively, we can turn the `downloadResources` function to a promise-returning function.